docs/src/pages/docs/crocks/Maybe.md

```diff
@@ -1438,6 +1448,14 @@ Just([])
 Just([ 'first', 'second', 'third' ])
   .chain(firstToMaybe(firstValue))
 //=> Just "first"
+
+Just(First('first'))
+  .chain(firstToMaybe)
+//=> Just "first"
+
+Just(First.empty())
```

Comment:
Should we also use an example using Last? What this will return ?

Reply:
Hey @HenriqueLimas this would throw an error `TypeError: firstToMaybe: First or First returning function required`, i'm not sure it makes sense unless you mean to want to show that?